spec/support/elasticsearch_helpers.rb

```diff
@@ -1,7 +1,9 @@
 # Only included in specs with elasticsearch: true
 module ElasticsearchHelpers
   def index_documents(resources)
-    Array.wrap(resources).each(&:index_to_elasticsearch_inline)
-    described_class.refresh_index
+    Array.wrap(resources).each do |resource|
+      resource.index_to_elasticsearch_inline
+      resource::SEARCH_CLASS.refresh_index
```

Comment:
There is no need to do this after every index and it will only slow us down. You only need to refresh the index once after everything has been indexed.